pkg/channel/fanout/fanout_message_handler.go

```diff
@@ -307,7 +307,7 @@ func (f *FanoutMessageHandler) makeFanoutRequest(ctx context.Context, message bi
 	)
 }
 
-type dispatchResult struct {
-	err  error
-	info *channel.DispatchExecutionInfo
+type DispatchResult struct {
+	Err  error
+	Info *channel.DispatchExecutionInfo
```

Comment:
I prefer if you create getters for these two: `Err()` and `Info()`

Reply:
Done added a New.. func and some gettrts, keeping it immutable for now.